### PR TITLE
feat[openapi]: Update swagger to 5.1.3

### PR DESCRIPTION
**Pull request by alexnorell**
Update bundles for swagger UI to 5.1.3 to support the latest OpenAPI standard.

Latest release from: https://github.com/swagger-api/swagger-ui

TODO:
- [ ] Add theme support for new UI elements
  Files in question are:
    - `static/css/swagger.css`
    - `static/css/swagger-dark.css`
    - `static/css/swagger-light.css`

Unthemed Schemas

### CODE REVIEW

**Comment by alexnorell** [comment]
Looks like the expand/collapse all buttons don't work for the schemas. Probably javascript stuff right?

Maybe in here?  https://github.com/odos-xyz/hugo-theme-relearn/blob/bbb6c8d8825f30d24bc31baec46bc95f200f8d98/static/js/theme.js#L321

**Comment by jdmsu4** [comment]
> Looks like the expand/collapse all buttons don't work for the schemas. Probably javascript stuff right?
> 
> Maybe in here?
> 
> https://github.com/odos-xyz/hugo-theme-relearn/blob/bbb6c8d8825f30d24bc31baec46bc95f200f8d98/static/js/theme.js#L321

They were working in my local testing, which version / environment were you seeing issues on?

**Comment by alexnorell** [comment]
> > Looks like the expand/collapse all buttons don't work for the schemas. Probably javascript stuff right?
> > Maybe in here?
> > https://github.com/odos-xyz/hugo-theme-relearn/blob/bbb6c8d8825f30d24bc31baec46bc95f200f8d98/static/js/theme.js#L321
> 
> They were working in my local testing, which version / environment were you seeing issues on?

I've got the branch checked out to 9622e2407d7fd7c4eda8d2a675bfe04cb8318a4a.

I created the file `content/_index.md` with the contents:

```yaml
---
title: Hey
---

{{< openapi src="https://api.odos.xyz/openapi.json" >}}
```

And then start the server from the root directory with:

```sh
hugo server -D
```

Transparent background is working but all the schemas are expanded on page load and the expand all/collapse all buttons at the top of the page don't affect them.

**Comment by alexnorell** [comment]
I think this is the function that handles the top expand/collapse all. And also does the collapse all on page load.
https://github.com/odos-xyz/hugo-theme-relearn/blob/bbb6c8d8825f30d24bc31baec46bc95f200f8d98/static/js/theme.js#L355C21-L366C34

**Comment by jdmsu4** [comment]
Ah I see, I will see if they can be collapsed by default rq

File: exampleSite/content/shortcodes/openapi/_index.en.md [hunk 1]
```diff
@@ -1,56 +0,0 @@
----
```
Comment (alexnorell) [inline exampleSite/content/shortcodes/openapi/_index.en.md]: Should this be deleted?

Comment (jdmsu4) [reply exampleSite/content/shortcodes/openapi/_index.en.md]: Accidentally removed the whole file instead of my changes, its back now

**Pull request merged**